src/EventStore.Core/ClusterVNode.cs

```diff
@@ -1583,6 +1583,15 @@ void ConfigureNode(IApplicationBuilder app) {
 					$"Unknown {nameof(options.Database.Transform)} specified: {options.Database.Transform}");
 		}
 
+		void StartNodeUnwrapException(IApplicationBuilder app) {
+			try {
+				StartNode(app);
+			} catch (AggregateException aggEx) {
+				// We only really care that *something* is wrong - throw the first inner exception.
+				throw aggEx.InnerException;
```

Comment:
There is Qodana warning about possible NRE. It's better to write `throw aggEx?.InnerException ?? aggEx` or add **when** filter to **catch** statement like `when (aggEx.InnerException is { } innerEx)`

Reply:
Another approach is to use list pattern matching:  `when (aggEx.InnerExceptions is [var innerEx, ..])`

Reply:
ah yeah, thanks :+1: